### PR TITLE
docs(community): adds governance pages

### PR DESCRIPTION
**Pull request by skim1420**

### CODE REVIEW

File: community/governance/index.md [hunk 1]
```diff
@@ -5,4 +5,125 @@ sidebar:
   nav: community
 ---
 
-* [SIGs](/community/governance/sigs)
+{% include toc %}
+
+The Spinnaker OSS project is run by a group of appointed members who hold specific roles, managed by the technical oversight committee. The steering committee manages the governance structure of the project, revisiting and refining it as needs evolve.
```
Comment (dorbin) [inline community/governance/index.md:10]: "is run by a group of appointed members"
Is this the steering committee? The way this first sentence sounds, the (steering committee?) is managed by the technical oversight committee.

File: community/governance/index.md [hunk 2]
```diff
@@ -5,4 +5,125 @@ sidebar:
...
+
+The broader community is organized around a set of special interest groups (SIGs) focused on particular areas of interest.
```
Comment (dorbin) [inline community/governance/index.md:12]: "...is organized *into* a set of" sigs?

Link from "special interest groups (SIGs)" to the SIGs section below?

File: community/governance/index.md [hunk 3]
```diff
@@ -5,4 +5,125 @@ sidebar:
...
+* Adam Jordens (Netflix)
+* Lars Wander (Google)
+* Rob Zienert (Netflix)
+* Travis Tomsu (Google)
+
+The technical oversight committee is initially seeded with core members from Netflix and Google. We will leave further details such as number of seats, qualifications, tenure, TBD for the time being.
+
+### Steering Committee
+
+The steering committee’s responsibility and function lies in the continual shaping of the governance structure to serve the project’s needs best.
```
Comment (dorbin) [inline community/governance/index.md:101]: "The steering committee is responsible for shaping governance to best serve the project's needs."

(This is the clearer, less wordy, less redundant way to say the same thing, but your way sounds more idiomatic of these sorts of things, so if you'd like the keep the language as is, feel free, except still move "best" to before "serve.")

File: community/governance/index.md [hunk 4]
```diff
@@ -5,4 +5,125 @@ sidebar:
...
+
+Duties:
+
+* Set governance structure of the project
+* Address questions or concerns from the community about project culture, structure, and clarifications of roles, responsibilities and duties.
+* Ratify new SIGs
```
Comment (dorbin) [inline community/governance/index.md:107]: Periods after all or after none.

File: community/governance/index.md [hunk 5]
```diff
@@ -5,4 +5,125 @@ sidebar:
...
+
+Logistics
+
+* [Agenda doc](https://docs.google.com/document/d/1HMdwvBPM4uRFqoeAd7eEkVWIC8dQP40zFavOE5Kq-Eg/edit)
+* Email: [sc@spinnaker.io](mailto:sc@spinnaker.io)
+
+Current SC members:
```
Comment (dorbin) [inline community/governance/index.md:114]: You've got a colon after this one, but not after any of the others on this page. What about using heading-level 4 for all.

File: community/governance/index.md [hunk 6]
```diff
@@ -5,4 +5,125 @@ sidebar:
...
+
+* Andy Glover (Netflix)
+* Matt Duftler (Google)
+* Peter Stout (Netflix)
+* Ruslan Meshenberg (Netflix)
+* Steven Kim (Google)
+
+The steering committee is initially seeded by Netflix and Google as the project matures and its structure stabilizes. We will leave further details such as number of seats, qualifications, tenure, appointing measures, TBD for the time being.
```
Comment (dorbin) [inline community/governance/index.md:122]: s/as/while/

File: community/governance/index.md [hunk 7]
```diff
@@ -5,4 +5,125 @@ sidebar:
...
+
+## SIGS
+
+SIGs (special interest groups) are created as community interest and demand around particular topics become self evident, and continue on as their relevance and needs do. SIGs and their leads are ratified and greenlit by the [steering committee](#steering-committee).
```
Comment (dorbin) [inline community/governance/index.md:126]: s/self evident/evident/

s/as their relevance and needs do/as needed/

**Review by dorbin — CHANGES_REQUESTED** [review]
The editorial stuff is all optional, @skim1420 , but I *am* confused by that first paragraph.

File: community/governance/index.md (hunk 1 above)
Comment (skim1420) [reply community/governance/index.md:10]: This refers to reviewers and approvers.

**Pull request merged**